### ISSUE TITLE
Using QFileDialog::getOpenFileName in QGoodWindow

### ISSUE THREAD

**Issue by Fabrice-Lainard-Programming** [antonypro/QGoodWindow#7]
Hello and thank you for this well-functioning library.
There is a problem when using QFileDialog::getOpenFileName from the main window: the dialog is displayed correctly but a **second window appears**, with the same title... 

To help you reproduce the problem, I have modified your example called 'ExempleMinimal' as follows:

```
MainWindow::MainWindow(QWidget *parent) : QGoodWindow(parent)
{
    setMargins(40,0,0,0);
    QPushButton *button = new QPushButton("&Download", this);
    this->setCentralWidget(button);
    connect(button,&QPushButton::clicked,this, [=]{
        QFileDialog::getOpenFileName(this, tr("TEST file"), "c:\\", tr("TEST (.txt)"));
    });
}
```
Thanks you !

**Comment by antonypro** [comment]
@Fabrice-Lainard-Programming: Hi! I've edited your code to make it compilable! ;)

Also i could reproduce the bug described, and found a potential solution!

As a temporary workaround you can remove these lines of code until i release a new version of **QGoodWindow**:

https://github.com/antonypro/QGoodWindow/blob/1bb87bf65198b419cd6212e693c8278c7633c973/QGoodWindow/QGoodWindow/QGoodWindowSource/qgoodwindow.cpp#L1983
https://github.com/antonypro/QGoodWindow/blob/1bb87bf65198b419cd6212e693c8278c7633c973/QGoodWindow/QGoodWindow/QGoodWindowSource/qgoodwindow.cpp#L1984

**Comment by antonypro** [comment]
This issue has been fixed in version 2.0. 

I'll close this issue for now, but if you believe it's not really solved, please inform me so i can reopen the issue.
